docs/NewProject.md

```diff
@@ -0,0 +1,44 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+     
+     ## Adding a new project
+Do the following when adding a new project:
+
+### For all projects
+1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
+2. Right-click on the project and select Properties.
+2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
+   - Currently .NET Framework 4.7.1 is the target. 
+3. In the build tab, set the following for both Debug and Release configurations:
+   1. "Warning level" to 4.
+   2. "Treat warnings as errors" to "All".
+
+### For *non-test* projects only
+1. Add the following NuGet packages in Visual Studio to enable signing and code analysis:
+   ```
+   Microsoft.VisualStudioEng.MicroBuild.Core
+   Microsoft.CodeAnalysis.FxCopAnalyzers
+   ```
+2. Close the solution and use your text editor to make the following changes to your `.csproj` file to properly configure the version and signing options:
+   1. Add the following line with the other `.cs` files):<br>
+   `<Compile Include="$(TEMP)\A11yInsightsVersionInfo.cs" />`
+   2. Add the following below the last ItemGroup:<br>
+   ```
+   <ItemGroup>
+      <DropSignedFile Include="$(OutDir)\[your project name].dll" />
+   </ItemGroup>
+   <Import Project="..\..\build\settings.targets" />
```

Comment:
>settings [](start = 32, length = 8)

This is correct for now. Once we remove digital signing, we can simplify it to include delaysign.targets in all projects, and remove settings.targets from the repo